src/openforms/forms/api/viewsets.py

```diff
@@ -613,18 +625,42 @@ def logic_rules_list(self, request, *args, **kwargs):
         summary=_("JSON schema"),
         description=_("Generate the JSON schema for a form."),
         parameters=[UUID_OR_SLUG_PARAMETER],
-        request=None,
-        responses={status.HTTP_200_OK: OpenApiTypes.OBJECT},
+        request=FormJsonSchemaOptionsSerializer,
```

Comment:
this endpoint is called via a `GET` operation, which typically doesn't have a request body. Instead, you want to document the GET params via the `parameters` kwarg, see here for example: https://github.com/open-formulieren/open-forms/blob/34c8023497c0bc49dab35a5e6729d4a6aff60e11/src/openforms/prefill/api/views.py#L22

This serializer approach for filter parameters is not great, but it's the best we have for the time being.